### DISCUSSION TITLE
Options API not available anymore?

### DISCUSSION THREAD

**Discussion by dennisprzytarski** [quasarframework/quasar#11916] [General - Components / Directives / etc] [answered]
Hey there, 
I'm going mad after searching the forums on the website and the issues and discussions sections here on GitHub. 
Is it just me or is there no option to use the options API anymore? 
I read in the forums, that it's going to stay, but the option is gone frome the CLI "create" command and it's not recognized by Quasar, when I use it (VS code doesn't colorize the script section the way it's doing it usually, the data is not being interpolated and ESLint doesn't complain about the "cool").

Any help would be appreciated. Thanks!

![Screenshot 2022-01-05 190535](https://user-images.githubusercontent.com/26872474/148266849-33df2a24-4379-41cd-bc8a-206ebeab5670.jpg)

**Comment by metalsadman** [comment, marked as answer]
you didn't wrap your JS code in a `<script>` tag.

**Reply by dennisprzytarski** [reply]
Wow, blind from sitting in front of code the whole day. Thanks a lot!
